main/lsp/wrapper.h

```diff
@@ -3,32 +3,45 @@
 
 #include "spdlog/spdlog.h"
 // has to come before the next spdlog include. This comment stops formatter from reordering them
-#include "lsp.h"
 #include "main/lsp/LSPMessage.h"
+#include "main/lsp/lsp.h"
 #include "spdlog/sinks/stdout_color_sinks.h"
 #include <string_view>
 namespace sorbet::realmain::lsp {
 
-class LSPOutputToVector;
+class LSPOutput;
+class LSPProgrammaticInput;
 class LSPConfiguration;
+
 class LSPWrapper final {
-    static const std::string EMPTY_STRING;
```

Comment:
No longer needed; the default constructor for `string_view` constructs a zero-length string view.